internal/controller/postgrescluster/controller.go

```diff
@@ -142,7 +142,7 @@ func (r *Reconciler) Reconcile(
 	// Issue Warning Event if postgres version is EOL according to PostgreSQL:
 	// https://www.postgresql.org/support/versioning/
 	currentTime := time.Now()
-	if postgres.ReleaseIsFinal(cluster.Spec.PostgresVersion, currentTime) {
+	if postgres.ReleaseIsFinal(int(cluster.Spec.PostgresVersion), currentTime) {
```

Comment:
What if we changed the `ReleaseIsFinal` func to take an int32?

Reply:
Done.